### PR TITLE
Allow install with PHP 8.0

### PR DESCRIPTION
**Pull request by caugner**
See #53.

### CODE REVIEW

**Comment by kaystrobach** [comment]
is there anything against having `^7.1 || ^8.0` imho this is clearer and would avoid future problems.

**Comment by caugner** [comment]
@kaystrobach IMO opinion `>= 7.1` is preferable because it avoids the current situation in the first place (for `9.0`).

**Comment by hanneskod** [comment]
Hi @caugner and thank you for taking the time to fix this!

I agree with @kaystrobach though, as we actually don't know if it is compatible with php 9.

Could you please also add `8.0` to `.travis.yml`.

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         ]
     },
     "require": {
-        "php": "^7.1",
+        "php": ">=7.1",
```
Comment (kaystrobach) [inline composer.json:23]:
```suggestion
        "php": "^7.1 || ^8.0",
```

**Comment by FabianKoestring** [comment]
Thanks! Will you tag a new release?

**Comment by hanneskod** [comment]
Yupp, good point!

**Pull request closed without merging**